ops/Dockerfile

```diff
@@ -4,10 +4,7 @@ LABEL maintainer="Jérôme Cosandey"
 LABEL maintainer="Nicolas Borboën"
 
 RUN mkdir -p /usr/src/app
-COPY data.json /usr/src/app
-COPY package.json /usr/src/app
-COPY package-lock.json /usr/src/app
-COPY index.js /usr/src/app
+COPY ./dev /usr/src/app
```

Comment:
We need to affine that : this mean that this might copy all the node_modules in the image and increase the image size — as far as possible I would like to keep the image size the smallest possible.